["```\nerror: error validating \"test.yml\": error validating data: [ValidationError(Deployment.spec.template.spec.containers[0]): unknown field \"named\" in io.k8s.api.core.v1.Container, ValidationError(Deployment.spec.template.spec.containers[0]): missing required field \"name\" in io.k8s.api.core.v1.Container]; if you choose to ignore these errors, turn validation off with --validate=false\n```", "```\nerror: error validating \"test.yml\": error validating data: ValidationError(Deployment.spec.template.spec.containers[0]): unknown field \"color\" in io.k8s.api.core.v1.Container; if you choose to ignore these errors, turn validation off with --validate=false\n```", "```\nkubectl get pods\n\nNAME                  READY STATUS       RESTARTS AGE\nflask-659c86495-vlplb 0/1   ErrImagePull 0        4s\n```", "```\nkubectl describe pod flask-659c86495-vlplb\n```", "```\nName: flask-659c86495-vlplb\nNamespace: default\nNode: minikube/192.168.99.100\nStart Time: Sat, 17 Mar 2018 14:56:09 -0700\nLabels: app=flask\n pod-template-hash=215742051\nAnnotations: <none>\nStatus: Pending\nIP: 172.17.0.4\nControlled By: ReplicaSet/flask-659c86495\nContainers:\n flask:\n Container ID:\n Image: quay.io/kubernetes-for-developers/flask:0.2.1\n Image ID:\n Port: 5000/TCP\n State: Waiting\n   Reason: ImagePullBackOff\n Ready: False\n Restart Count: 0\n Environment: <none>\n Mounts:\n /var/run/secrets/kubernetes.io/serviceaccount from default-token-bwgcr (ro)\nConditions:\n Type Status\n Initialized True\n Ready False\n PodScheduled True\nVolumes:\n default-token-bwgcr:\n Type: Secret (a volume populated by a Secret)\n SecretName: default-token-bwgcr\n Optional: false\nQoS Class: BestEffort\nNode-Selectors: <none>\nTolerations: <none>\n```", "```\ndocker pull quay.io/kubernetes-for-developers/flask:0.2.1\n```", "```\nError response from daemon: manifest for quay.io/kubernetes-for-developers/flask:0.2.1 not found\n```", "```\nNAME                          READY STATUS           RESTARTS AGE\nflask-6587cb9b66-zzw8v        1/2   CrashLoopBackOff 2        1m\nredis-master-75c798658b-4x9h5 1/1   Running          0        1m\n```", "```\nkubectl describe pod flask-6587cb9b66-zzw8v\n```", "```\nkubectl logs flask-6587cb9b66-zzw8v\n\nError from server (BadRequest): a container name must be specified for pod flask-6587cb9b66-zzw8v, choose one of: [jaeger-agent flask] or one of the init containers: [init-myservice]\n```", "```\nkubectl logs flask-6587cb9b66-zzw8v -c flask\n\npython3: can't open file '/opt/exampleapp/exampleapp': [Errno 2] No such file or directory\n```", "```\ndocker pull quay.io/kubernetes-for-developers/flask:latest\n```", "```\ndocker inspect quay.io/kubernetes-for-developers/flask:latest\n```", "```\ndocker run -it --entrypoint=/bin/sh \\\nquay.io/kubernetes-for-developers/flask:latest -i\n```", "```\nkubectl run -i --tty interactive-pod \\\n--image=quay.io/kubernetes-for-developers/flask:latest \\\n--restart=Never --command=true /bin/sh\n```", "```\nError from server (AlreadyExists): pods \"interactive-pod\" already exists\n```", "```\nkubectl get pods -a\n\nNAME                          READY STATUS           RESTARTS AGE\nflask-6587cb9b66-zzw8v        1/2   CrashLoopBackOff 14       49m\ninteractive-pod               0/1   Completed        0        5m\nredis-master-75c798658b-4x9h5 1/1   Running          0        49m\n```", "```\nkubectl delete pod interactive-test\n```", "```\n/bin/sh: can't open '/some/script'\n```", "```\nno endpoints available for service\n```", "```\n{\n  \"kind\": \"Status\",\n  \"apiVersion\": \"v1\",\n  \"metadata\": {\n\n  },\n  \"status\": \"Failure\",\n  \"message\": \"no endpoints available for service \\\"flask-service\\\"\",\n  \"reason\": \"ServiceUnavailable\",\n  \"code\": 503\n}\n```", "```\nhttp://localhost:8001/api/v1/proxy/namespaces/default/services/flask-service\n```", "```\nkubectl describe service flask-service\n```", "```\nkubectl describe deploy flask\n```", "```\nkubectl get pods -l app=flaskapp -o wide\n```", "```\nNo resources found.\n```", "```\nNAME                          READY STATUS   RESTARTS AGE\nflask-6bc4b4c8dc-cm6gx        0/2   Init:0/1 0        7m\nredis-master-75c798658b-p4t7c 1/1   Running  0        7m\n```", "```\nkubectl logs flask-6bc4b4c8dc-cm6gx init-myservice\n```", "```\nError from server (BadRequest): container \"init-myservice\" in pod \"flask-6bc4b4c8dc-cm6gx\" is waiting to start: PodInitializing\n```", "```\nkubectl describe pod flask\n```", "```\nMountVolume.SetUp failed for volume \"config\" : configmaps \"flaskConfig\" not found\n```"]